### PR TITLE
Do not pass string in mastertag when should be json

### PR DESCRIPTION
**Pull request by berlintam**
this PR formats the json of the mastertag properly.

### CODE REVIEW

**Comment by berlintam** [comment]
@salimhb I have a failing spec in the `zanox_integration_spec` and I can't for the life of me figure out why. Can you take a quick look?

**Comment by salimhb** [comment]
@berlintam the other PR is merged now. merge master again into this branch and let's see how it looks

**Comment by salimhb** [comment]
I guess now it's more obvious what's wrong, right?

File: spec/handler/zanox_spec.rb [hunk 1]
```diff
@@ -96,7 +105,6 @@ def env
 
       it 'will display the correct tracking events' do
         expect(subject).to include 'window._zx.push({"id": "12345678D2345"});'
-        expect(subject).to include "var zx_category = 'Sewing';\nvar zx_identifier = '234';\nvar zx_amount = '5.90';"
```
Comment (salimhb) [inline spec/handler/zanox_spec.rb]: why is this deleted?

Comment (salimhb) [inline spec/handler/zanox_spec.rb]: to_json wraps the values with double quotes instead of single quotes. so this should stay, but the expected value should use double quotes

**Comment by berlintam** [comment]
@salimhb it's green! :sweat_smile:

**Comment by salimhb** [comment]
:+1:

**Comment by DonSchado** [comment]
:+1:
